docs/core/tools/dotnet-nuget-push.md

```diff
@@ -30,6 +30,17 @@ The `dotnet nuget push` command pushes a package to the server and publishes it.
 
 The command pushes an existing package. It doesn't create a package. To create a package, use [`dotnet pack`](dotnet-pack.md).
 
+### Hierarchical folder structure
+
+This command can store packages in a hierarchical folder structure, which is recommended to optimize performance. It stores packages in a hierarchical folder structure when publishing to a local folder (feed), like `nuget add` does, if there already is at least one package in the feed that is in a hierarchical folder structure. If the feed has a hierarchical folder structured package already in it, `dotnet nuget push` respects that structure. So, if you want to publish to a local feed using the .NET CLI instead of the NuGet CLI:
+
+* Before you publish the first package, go to the local cache folder, that is, global packages at *%userprofile%.nuget\packages*, and select the root folder of a package id. It can be any package that is not part of a framework, like .NET standard or ASP.NET.d
```

Comment:
```suggestion
* Before you publish the first package, go to your global packages folder folder, at *%userprofile%.nuget\packages*, and select the root folder of a package id. It can be any package that is not part of a framework, like .NET standard or ASP.NET.
```

We try not to refer to the global packages folder as cache since there's no eviction policy.

Feel free to tweak if you'd like as long as we don't refer to as is cache.